src/lua_api_things.c

```diff
@@ -65,6 +65,14 @@ static int lua_delete_thing(lua_State *L)
 
     if (thing->class_id == TCls_Creature)
```

Comment:
only if it's a creature? I suppose if you delete a picked up object or something you'd also want to remove it from hand

Reply:
ok, made it work for non-creatures too, but need a lua test script